app/models/measure_type.rb

```diff
@@ -48,6 +49,10 @@ def abbreviation
     'Control'
   end
 
+  def cds_proofs_of_origin?
+    CDS_PROOFS_OF_ORIGIN.include? id
```

Comment:
I'm wondering if this should be source_id?

Reply:
hmm - or possibly `measure_type_id` - I'll change it :+1:

Reply:
Nah - this needs to be `id` - `resource_id` is kinda internal and there isn't a `measure_type_id`